modules/serverless-logic-asyncapi-consuming-event.adoc

```diff
@@ -1,15 +1,15 @@
 // Module included in the following assemblies:
-//
+// * serverless-logic/serverless-logic-managing-events/serverless-logic-configuring-asyncapi
 
 :_mod-docs-content-type: PROCEDURE
-[id="serverless-logic-asyncAPI-consuming-event_{context}"]
+[id="serverless-logic-asyncapi-consuming-event_{context}"]
 = Consuming an event
 
 To enable seamless event consumption, define the Kafka server, subscription channel, and a function that consumes the event, and then invoke the function in your workflow. When your workflow reaches the function, it pauses and waits for the specific event from the Kafka topic before continuing.
 
 .Prerequisites
-* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
-* You have created your {ServerlessLogicProductName} project.
+
+* You have access to an {ServerlessLogicProductName} project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
 * You have created the `asyncAPI.yaml` file and your project `classpath`.
```

Comment:
`classpath` ? Do you mind elaborating why that word is used? How do I create my project classpath?

Reply:
Should `asyncAPI.yaml` be empty or do I already have it because I followed some other procedure?

Reply:
I guess a small bug, the path should be from root of OSL project - like `WF_ROOT/specs`